src/lib/i18n.ts

```diff
@@ -0,0 +1,39 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+
+export enum Language {
+  EnUS = 'en-US',
+  EnGB = 'en-GB',
+  Pa = 'pa',
+}
+
+export const createTranslations = (
+  translations: Record<string, unknown>, // : {
+  //   language: string,
+  //   translationSet: { phrase: string, translation: string },
+  // },
+  namespace: string,
+) => {
+  i18n.setDefaultNamespace( namespace )

```

Comment:
Why does this need to be done?

Reply:
It does not need to be. We have a few options:

1) Use only one namespace for all translations (e.g. `translations`), then reference them as `t(phraseToTranslate)`
2) Use the component name as the translation (e.g. `search`), then reference them as `t(search.phraseToTranslate)`
3) Use the component name as the translation (e.g. `search), set default namespace in that component, then reference as `t(phraseToTranslate)`

I'm more than happy to not use namespaces, it also cuts down on the code, and I don't see how we're going to be doing lazy loading of namespace translations just yet. But it's an idea in case you want to do lazy loading in the future to break these translations into their own name spaces.

Reply:
Doesn't the current option presented set the namespace globally? This is my confusion

Reply:
Oh yes you're correct that it's global, that may cause issues I suppose, easiest is to load all translations once in a single namespace or to reference the namespace in each component in the `t(namespace:phrase)` convention